release-tools/src/main/java/org/springframework/data/release/build/MavenBuildSystem.java

```diff
@@ -277,18 +275,72 @@ public ModuleIteration prepareVersion(ModuleIteration module, Phase phase) {
 		return module;
 	}
 
-	/*
-	 * (non-Javadoc)
-	 * @see org.springframework.data.release.build.BuildSystem#deploy(org.springframework.data.release.model.ModuleIteration)
+	/**
+	 * Perform a {@literal nexus-staging:rc-open} and extract the stagingProfileId from the results.
 	 */
 	@Override
+	public void open() {
+
+		try {
+			CommandLine arguments = CommandLine.of(Goal.goal("nexus-staging:rc-open"), //
+					profile("central"), //
+					of("-s " + properties.getSettingsXml()), //
+					arg("stagingProfileId").withValue(properties.getMavenCentral().getStagingProfileId()), //
+					arg("openedRepositoryMessageFormat").withValue("'" + REPO_OPENING_TAG + "%s" + REPO_CLOSING_TAG + "'"));
+
+			mvn.execute(BUILD, arguments);
+
+			String rcOpenLogfile = "mvn-" + BUILD.getName() + "-nexus-staging.rc-open.log";
```

Comment:
Log files are only produced when the Maven build config is configured to do so. For now it's okay, but let me come up with a better mechanism that doesn't fail if log files are not configured.